WordPressKit/PluginDirectoryEntry.swift

```diff
@@ -83,16 +83,16 @@ extension PluginDirectoryEntry: Codable {
         lastUpdated = try? container.decode(Date.self, forKey: .lastUpdated)
         rating = try container.decode(Int.self, forKey: .rating)
 
-        let icons = try? container.decodeIfPresent([String: String].self, forKey: .icons)
+        let icons = ((try? container.decodeIfPresent([String: String].self, forKey: .icons)) as [String : String]??)
```

Comment:
is there a specific reason for casting to `[String: String]?` ? `decodeIfPresent` should return a `[String: String]` value or `nil`, depending if that value exists (and is not nil) or not, so if we leave this statement unchanged, can the following just be 
 `icon = icons?["2x"].flatMap(URL.init(string:))`?

or we could also:

``if let icons = try container.decodeIfPresent([String: String].self, forKey: .icons) {
            icon = icons["2x"].flatMap(URL.init(string:))
 }``

Reply:
No there is not. I let Xcode do the migrating and overlooked how odd that change really was. Correcting it now :)